lib/templates.js

```diff
@@ -72,6 +74,23 @@ templates.add('send-feedback', `
     padding: 12px 15px;
   }
 
+  #closebtn {
+    border: 2px solid #D9534F;
+    background-color: #D9534F;
+    display: inline-block;
+    color: #fff;
+    width: auto;
+    line - height: 9 px;
+    float: right;
+    cursor: pointer;
+    border - radius: 4 px;
+    transition: background - color, opacity 0.1 s ease - in ;
+    margin - bottom: 6 px;
+    text - align: center;
+    text - overflow: ellipsis;
+    padding: 12 px 15 px;
+  }
```

Comment:
We already have a css written for the button. So all you had to do here is change the background color. No need to restyle the already styled button here.

Reply:
I see thanks for the comments. I really appreciate you spending your time to explain such elementary stuff.